### DISCUSSION TITLE
Hydroxylation of proline as varmod

### DISCUSSION THREAD

**Discussion by GOzorun** [vdemichev/DiaNN#1213] [Q&A]
Hi,

I tried including hydroxylation of proline and lysine as varmod in additional options, it seemed to recognize the command as the log mentions "Modification UniMod:35 with mass delta 15.9949 at PK will be considered as variable". However, no peptides were identified with this mod. 

Then I found this in the documentation "The modifications currently supported by the deep learning predictor are: C(cam), M(ox), N-term acetyl, N/Q(dea), S/T/Y(phos), K(-GG), nK(mTRAQ) and nK(TMT)."

Is it correct that I cannot search for hydroxylation of proline and lysine using DIA-NN? Please could you let me know if there is an alternative way as it would vastly improve my research.

Thank you.

**Comment by vdemichev** [comment]
Hi,

The in silico predictor was not trained on these, but it will still work :)

If nothing is identified, please share the logs (all, including in silico lib generation) and I take a look?

Best,
Vadim

**Reply by GOzorun** [reply]
[report.log.txt](https://github.com/user-attachments/files/17409760/report.log.txt)
[report-lib.log.txt](https://github.com/user-attachments/files/17409761/report-lib.log.txt)
Thanks Vadim, I've attached the logs.

**Reply by vdemichev** [reply]
Same --var-mod declared twice with different amino acid specificities. DIA-NN then considers the second declaration a 'duplicate' and removes it, so only putting the mod on methionines. The solution would be to just declare as --var-mod UniMod:35,15.994915,MPK

**Reply by GOzorun** [reply]
Ah I see, thank you very much!
